src/docs/product/projects/project-details/index.mdx

```diff
@@ -4,47 +4,140 @@ sidebar_order: 1
+<div style="position: relative; padding-bottom: calc(61.72148355493352% + 41px); height: 0; width: 100%">
+  <iframe
+    src="https://demo.arcade.software/ZEOZDNURwYhmkqdQ76ge?embed"
+    frameborder="0"
+    loading="lazy"
+    webkitallowfullscreen
+    mozallowfullscreen
+    allowfullscreen
+    style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;color-scheme: light;"
+    title="Project Details -> Create Alert"
+  ></iframe>
+</div>
 
-## Releases
+## View Latest Alerts
 
-You can use the Project Details page to assess how often you release new versions of your application:
+The **Latest Alerts** section in the right sidebar shows the 3 latest triggered alerts. Click on an alert to go to the **Alert Details** page. The selected time range does not affect the displayed alerts.
+
+<div style="position: relative; padding-bottom: calc(61.72148355493352% + 41px); height: 0; width: 100%">
+  <iframe
+    src="https://demo.arcade.software/2pJrK9pQvEwpVRseSxbz?embed"
+    frameborder="0"
+    loading="lazy"
+    webkitallowfullscreen
+    mozallowfullscreen
+    allowfullscreen
+    style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;color-scheme: light;"
+    title="Project Details -> Latest Alerts"
+  ></iframe>
+</div>
+
+## View Latest Releases
+
+You can use the **Project Details** page to assess how often you release new versions of your application:
 
 - Number of releases compared to the previous period in the top card
 - Release lines overlaid on top of graphs
 - Most recent five releases in the **Latest Releases** section of the sidebar. You can click one of these to navigate to the [Release Details](/product/releases/health/release-details/) page. The selected time range does not affect the displayed releases.
 
-## Triage
+## Triage Top Issues
 
-Use Project Details to view issues associated with this project and to access them for triage. You can navigate to the **Issue Details** page by clicking either **View All Issues** or the **Open in Issues** button located at the top of the **Frequent Unhandled Issues** listed on the page. The issues in the table are sorted by the number of events in the selected time range.
+To help you triage issues, the **Project Details** page shows you the 5 most frequent unhandled issues, sorted by the number of events in the selected time range. Click on an issue to navigate to the **Issue Details** page.
 
-## Modify Project Settings
+<div style="position: relative; padding-bottom: calc(54.296875% + 41px); height: 0; width: 100%">
```

Comment:
Same comment about starting the arcade on step 2.

Also same about not being able to tell what step 4 was for.